webapp/components/test-runs-query.html

```diff
@@ -205,14 +206,25 @@
           this.setProperties(batchUpdate);
         }
 
-        computeResultsRangeMessage(from, maxCount) {
-          if (from) {
-            var diff = Math.floor((Date.now() - from) / 86400000);
-            return `Showing test runs from the last ${diff} days`;
+        computeResultsRangeMessage(sha, productSpecs, from, to, maxCount) {
+          if (sha) {
+            return `Showing results from revision ${sha}`;
+          } else if (from) {
+            const max = to || Date.now();
+            var diff = Math.floor((max - from) / 86400000);
+            const shortDate = date => date.toLocaleDateString(navigator.language, {
+              month: 'short',
+              day: 'numeric',
+            });
+            return to
+              ? `Showing test runs from ${shortDate(from)} to ${shortDate(to)}`
+              : `Showing test runs from the last ${diff} days`;
           } else if (maxCount) {
             return `Showing ${maxCount} test runs per product`;
+          } else if (this.computeIsDefaultProducts(productSpecs)) {
+            return 'Showing the latest test runs for the default product set';
           }
-          return null;
+          return `Showing the latest test runs for ${productSpecs.join(', ')}`;
```

Comment:
Is it possible to turn this into more human friendly strings, like "chrome[experimental]" -> "Chrome Dev"? I suspect answer is not yet, so follow up is fine :)

Reply:
Yah will follow-up